### ISSUE TITLE
Incorrect heading level for "Manage your health online"

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12079]
## Describe the defect
On VAMC System landing pages, there is a heading for "Manage your health online" that is currently an H3 but should be an H2 to properly fit the heading structure of the page.

May be done at the same time as #17436 which is currently blocked by Health Chat/Health Connect CLP being published.

## AC / Expected behavior

- [ ] Update "Manage your health online" heading to be an H2
- [ ] Accessibility review 

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/209154902-58e84f38-bb38-4220-9639-ac798ef4cb9b.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
No this is an existing issue on the VAMC product

**Comment by briandeconinck** [comment]
This issue was identified during the Collaboration Cycle Staging Review for Lovell VAMC on 2/28/2023. Applicable Collaboration Cycle labels have been added to this ticket for tracking and dashboard purposes, rather than creating a duplicate ticket.

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team Ticket # [64837](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64837)

**Comment by laflannery** [comment]
Removing CC labels because ticket was recreated

**Comment by xiongjaneg** [comment]
This could be done at the same time as adding health chat and health connect. @xiongjaneg to find that ticket.

**Comment by Agile6MSkinner** [comment]
Being handled in 17436. Closing.
